testing/model-mesh/5-dump-and-clear-prom-db.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./run_toolbox.py cluster capture_environment
```

Comment:
I don't think this one does anything interesting for you
saving the full state of the cluster is done in the destroy step

Reply:
Solved, I think, thx